### ISSUE TITLE
Improve design of GH pages

### ISSUE THREAD

**Issue by ifraixedes** [ifraixedes/hugo-resources#4]
Github pages are generated with the Hugo site which is in root directory of this repository and it contains the documentation of the hugo resources which this repository contains.

`makefile` has `gen-gh-pages` target which build the hugo site into `gh-pages` subdirectory with the content to publish in `gh-pages` orphan branch.

So far there isn't a design at all, so it will be nice to have a better design for it.

This issues is generated after I worked in #2 so it's required to merge it before starting with this
